sorc/ncep_post.fd/PROCESS.f

```diff
@@ -43,30 +43,31 @@
 !!       FIXED    - POST FIXED FIELDS.
 !!     LIBRARY:
 !!       COMMON   - OUTGRD
-!!     
+!!
 !!   ATTRIBUTES:
 !!     LANGUAGE: FORTRAN
 !!     MACHINE : CRAY C-90
 !!
       SUBROUTINE PROCESS(kth,kpv,th,pv,iostatusD3D)
-
 !
 !----------------------------------------------------------------------------
-!     
+!
+      use mpi, only: mpi_wtime
```

Comment:
@DusanJovic-NOAA I saw other UPP routines using 'include "mpif.h" '. Are there any differences of these two statements on runtime performance?

Reply:
No, there shouldn't be any difference in runtime performance.

Reply:
Thanks!